.travis.yml

```diff
@@ -1,17 +1,36 @@
 language: node_js
-
 node_js:
   - 8.9.4
 
+branches:
+  only:
+    - master
+    - /v\d+\.\d+\.d+$/
+
 cache:
-  directory:
-    - studio-frontend/node_modules/
+  directories:
+    - "node_modules"
```

Comment:
I don't think this will matter much at the end of the day, but I think it's slightly better to cache `~/.npm` vs. `node_modules` even though I know Travis' official docs say to use `node_modules`

See 

* [SO article](https://stackoverflow.com/a/42523517/5225575)
* [GH Issue + linked Twitter thread](https://github.com/semantic-release/cli/issues/125)